assets/js/base/utils/pick.ts

```diff
@@ -0,0 +1,8 @@
+export const pick = < Type >( object: Type, keys: string[] ): Type => {
```

Comment:
Would it be possible to include some comments about the function?  It would be helpful to have a better understanding of what it is specifically accomplishing.